Favoriterestaurantsresults

Kimberly-Rodriguez opened the pull request:
creating a new favorite restaurant results component

Kimberly-Rodriguez committed:
connecting favrestaurants to userId

Kimberly-Rodriguez committed:
Merge remote-tracking branch 'origin/develop' into favoriterestaurantsresults

Kimberly-Rodriguez committed:
updating route for fav-result and passing it a currentUser props and adding a useeffect to pass the inial object to map it

johndavis92790 reviewed (commented):
I will get the photo and the google rating to work on my end and then I will push up the changes, thanks!

johndavis92790 committed:
Compartmentalize the favoriting icon.
- This allows the heart icon to be used anywhere in the app.
- Add this to the single result page, multiple result page and favorites page.
- Refactor code in a few places.
- Delete unused files from other tests.
- Rename a few files to better match their functionality.
